### PR TITLE
Fix example reference in resource groups doc

### PR DESCRIPTION
**Pull request by mfzarko**

### CODE REVIEW

**Review by mosabua — APPROVED** [review]
Nice catch and thanks for the fix.

File: docs/src/main/sphinx/admin/resource-groups.rst [hunk 1]
```diff
@@ -145,7 +145,7 @@ There are four selectors, that define which queries run in which resource group:
   These are placed in a dynamically-created sub-group under the ``global.pipeline.tools`` group. The dynamic
```
Comment (phd3) [inline docs/src/main/sphinx/admin/resource-groups.rst:145]: `global.adhoc` group

Comment (phd3) [inline docs/src/main/sphinx/admin/resource-groups.rst:145]: `hi-pri` to `hipri` on previous line

File: docs/src/main/sphinx/admin/resource-groups.rst [hunk 2]
```diff
@@ -145,7 +145,7 @@ There are four selectors, that define which queries run in which resource group:
...
   sub-group are created based on the named variable ``toolname``, which is extracted from the in the
```
Comment (phd3) [inline docs/src/main/sphinx/admin/resource-groups.rst:146]: modification and grammar fix: (cc @mosabua)

```
The dynamic sub-groups are created based on the values of named variables ``toolname`` 
and ``user``. The values are derived from the source regular expression and the query user 
respectively.
```

**Review by phd3 — COMMENTED** [review]
Thanks for catching this. The same bullet seems to have some other outdated references too, can you please fix them too for consistency?

File: docs/src/main/sphinx/admin/resource-groups.rst (hunk 2 above)
Comment (mfzarko) [reply docs/src/main/sphinx/admin/resource-groups.rst]: Done

**Comment by mfzarko** [comment]
Updated per @phd3 suggestions. Thanks you!

**Comment by phd3** [comment]
Unrelated failure, looks similar to #4936 

```
io.trino.tests.hive.TestHiveStorageFormats.testInsertIntoPartitionedTable [StorageFormat{name=AVRO, properties={}, sessionProperties={}}] (Groups: storage_formats)
```

**Comment by phd3** [comment]
Merged after a minor fixup, thanks for improving the documentation @mfzarko !

**Pull request merged**